lib/Transforms/IPO/PassManagerBuilder.cpp

```diff
@@ -617,7 +617,6 @@ void PassManagerBuilder::populateModulePassManager(
   addExtensionsToPM(EP_Peephole, MPM);
   MPM.add(createCFGSimplificationPass());
   MPM.add(createDxilLoopDeletionPass()); // HLSL Change - try to delete loop again.
-  MPM.add(createInstructionCombiningPass());
```

Comment:
might want to comment it out (instead of deleting) with a note that it's HLSL specific change..